### PR TITLE
Feature/aboutus active function Created

### PR DESCRIPTION
**Pull request by bosilf**
I have changed the function to remove the active tab when clicked twice to show the intro content. 
I have created a function (introContent) that will show the first content when the page is entered.
I edited the structure of the code for an easier navigation.

How it should work: 
When entering the aboutus.html page you should see the introContent which is a welcome message about the team; a title, species, paragraph, image, and an info bar about the team. 

In the sidebar, only the names of the team members should show, when a team member tab is clicked the content on the screen will change to the active tab. When the same tab is pressed twice in a row the content on the page changes back to the introContent.

It should be responsive to all screen sizes.

### CODE REVIEW

**Review by Endorze — APPROVED** [review]
Looks gud! 🔥

File: scripts/aboutus.js [hunk 1]
```diff
@@ -94,144 +110,128 @@ const teamMembers = [
...
-            speciesWrapper.appendChild(species);
-
-
     const teamParagraph = document.createElement("p");
     teamParagraph.classList.add("text__content", "paragraph");
-    teamParagraph.textContent = person.aboutText; 
+    teamParagraph.textContent = content.aboutText; 
 
     teamSection.appendChild(teamImage);
+        teamImage.appendChild(imageObject)
```
Comment (maysunktong) [inline scripts/aboutus.js:215]: I enjoy the .appendChild heaven ☁️

File: scripts/aboutus.js [hunk 2]
```diff
@@ -94,144 +110,128 @@ const teamMembers = [
...
+            }
         });
-
         tabsContainer.appendChild(tab);
     });
 };
 
-
 const setActiveTab = (activeTab) => {
     document.querySelectorAll(".tab").forEach((tab) => tab.classList.remove("active"));
```
Comment (maysunktong) [inline scripts/aboutus.js:138]: in my humble opinion, you can query two classes at the same time,
.querySelect(".tab.active")

if one element has this active, .remove() the class.

**Review by maysunktong — APPROVED** [review]
You have written a nice detailing info for your PR which tells reviewers what they are expected to see and code review. 

Look good to me 🥳

**Pull request merged**